src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -654,6 +552,9 @@ def set_environment_variables(self):
     """Sets environment variables needed by afl."""
     # Tell afl_driver to duplicate stderr to STDERR_FILENAME.
     # Environment variable names and values that must be set before running afl.
+    environment.set_value(constants.AFL_MAP_SIZE_ENV_VAR, 8000000)
```

Comment:
Why do we need to set this?

Reply:
since 2 days not anymore. before this was necessary because with non-colliding coverage it is hard to guess how large the map size is for a target. now there is an inherent large default plus it is tried to gather the map size in a pre-start.

Reply:
Got it.

Reply:
Already did, look, it is marked "Outdated" :)